.gitignore

```diff
@@ -78,4 +78,4 @@ tmp*
 /clients/python/artifacts/*
 /clients/python/client/build/*
 /clients/python/client/osparc/data/openapi.json
-/clients/python/test/e2e/pyproject.toml
+/clients/python/test/e2e/pytest.ini
```

Comment:
A `pytest.ini` file is generated for every (server, client) pair. That one holds the entire configuration needed to run the tests. This makes it easy to reproduce locally by simply downloading this file from the artifacts. But it should not be stored in the repo because it holds the key, secret pairs